src/Reproject.jl

```diff
@@ -1,8 +1,14 @@
 module Reproject
 
-using FITSIO, WCS
+using FITSIO, WCS, Interpolations, SkyCoords
+using SkyCoords: lat,lon
 
 include("parsers.jl")
 include("utils.jl")
+include("core.jl")
+
+export
+    reproject
 
 end # module
+
```

Comment:
Again: you don't need to add an extra empty line, just make sure that the last line is complete, i.e., it's ended by a newline character